src/windows/FileProxy.js

```diff
@@ -751,7 +751,13 @@ module.exports = {
         if (!fs || !validName(path)){
             fail(FileError.ENCODING_ERR);
             return;
-        }           
+        }
+	    
+        //check trailing "/"
```

Comment:
this actually removes a trailing slash, correct?

Reply:
Correct.  I submitted a (dup) JIRA report that explains why this slash causes a failure downstream: https://issues.apache.org/jira/browse/CB-12945 .